backend/kernelCI_app/management/commands/notifications.py

```diff
@@ -482,7 +487,65 @@ def discard_sent_reports(
     return key_set_copy, prop_map_copy
 
 
+def get_build_issues_from_checkout(
+    *, checkout_ids: list[str]
+) -> tuple[dict[str, list[CheckoutIssue]], dict[str, list[str]]]:
+    checkout_issue_record = get_issues_summary_data(checkout_ids=checkout_ids)
+
+    checkout_issues: dict[str, list[dict]] = defaultdict(list)
```

Comment:
If I understood correctly, this variable is a map for getting all the issue information based on its id. So I don't think that `checkout_issues` is a great name for it, made me think that the checkout_id should be the key for this.
```suggestion
    issues_map: dict[str, list[dict]] = defaultdict(list)
```

Reply:
`issue_checkouts_map` makes me think of a map from issues to a list of checkouts... This var maps from issue_id to a list of issues